cloudbuild.yaml

```diff
@@ -0,0 +1,4 @@
+steps:
+- name: "gcr.io/cloud-builders/gcloud"
+  args: ["app", "deploy"]
+timeout: "1600s"
```

Comment:
That's quite a generous timeout (26 minutes), but not really a problem I suppose